docs/data-sources/groups.md

```diff
@@ -28,8 +33,13 @@ data "twingate_groups" "foo" {
 
 - `groups` (Attributes List) List of Groups (see [below for nested schema](#nestedatt--groups))
```

Comment:
Is there a difference between

```
data "twingate_groups" "foo" {
  groups = {
    name = "test"
    is_active = true
  }
}  
``` 

and

```
data "twingate_groups" "foo" {
  name = "test"
  is_active = true
}  
```

Do we still need the `groups` attribute? If so should we just add the filters to it?

Reply:
We dont have this extra hierarchy in other datasources?
Personally, I would rather have a breaking change and get rid of it  than have this weird duplication of functionality

Reply:
I see we have this pattern in all our datasources...

Reply:
fixed